src/hu_nmt/data_augmentator/augmentators/subject_object_augmentator.py

```diff
@@ -348,6 +413,22 @@ def find_augmentable_candidates(hun_graphs: List[DependencyGraphWrapper], eng_gr
 
         return augmentation_candidate_translations
 
+    @staticmethod
+    def find_separate_candidates(hun_graphs: List[DependencyGraphWrapper], eng_graphs: List[DependencyGraphWrapper], with_progress_bar: bool = False) -> Dict[str, List[TranslationGraph]]:
```

Comment:
`separate` in the function name I find a bit confusing. Could we just name it `find_candidates`?